public/Lab-Files/Lesson03/Assignments/blog-page.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Johnny Lopez's Blog</title>
+    <link rel="stylesheet" href="blog-page.css" type="text/css" />
+</head>
+
+<body>
+    <a href="../../../index.html">Go Back</a>
+
+    <header>
+        <h1>The Misadventures of a Budding Web Developer!</h1>
+    </header>
+
+    <nav>
+        <div class="topnav" id="myTopnav">
+            <a href="#home">Home</a>
+            <a href="#about">About</a>
+            <a href="#archives">Archives</a>
+            <a href="#social">Social Media</a>
+        </div>
+    </nav>
+
+    <main>
+        <h2>My First Blog Post</h2>
+        <time>March 13th, 2017</time>
+        <p>This is my first blog post. It was a fun process to come up with the layout for my first blog post. I started it by looking up other blogs to get some inspiration as to how I wanted it to look. I also took a look at all the HTML elements I could
+            use to struture the blog.</p>
+        <p>I sketched out they layout in one of my notebooks. I add a picture of it down below. I also came up with the main title and some other element that I would be using as well. Next I would have to decided on the styling, which I completed with some
+            CSS. I am looking forward to my continued learning on being a web developer. That way I can continue to make awesome webpages.</p>
+        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Facilis, aliquid eos quae eligendi dolor assumenda aperiam consequuntur quidem minus debitis recusandae odit ab doloremque ratione iure voluptate praesentium aspernatur excepturi perferendis
+            deleniti dolore accusantium culpa quos expedita doloribus harum optio maiores blanditiis itaque alias! Sunt laborum animi est omnis quis.</p>
+
+        <img class="pic" src="IMG_0775.JPG" alt="A notebook with sketch of a website" style="width:400px;hight:auto" />
+    </main>
+
+    <aside class="top">
+        <h4>About Me</h4>
+        <p>I am currently a web developer student at San Antonio Codeing Academy. I am also a Chief Workshop Manager at Build A Bear Workshop. Time Managerment is a must for me and my busy schedule.</p>
+    </aside>
+
+    <aside class="bottom">
+        <h4>Like the Blog? Check out more here:</h4>
+        <ul>
+            <li><a href="#">Blog 2</a></li>
+            <li><a href="#">Blog 3</a></li>
+            <li><a href="#">Blog 4</a></li>
+        </ul>
+    </aside>
+
+    <footer>
+        <p>Posted by: Johnny Lopez</p>
+        <p>Contact information: <a href="johnnyll137@gmail.com">johnnyll137@gmail.com</a></p>
+    </footer>
+</body>
+
+</html>
```

Comment:
I really like how you organized this assignment. It's awesome that you sketched it out and then started coding!